MailCore/API/MailApiFetcher/MailApiFetcher+Common.swift

```diff
@@ -98,6 +98,14 @@ public extension MailApiFetcher {
         try await perform(request: authenticatedRequest(.quotas(mailbox: mailbox.mailbox, productId: mailbox.hostingId)))
     }
 
+    func externalMailFlag(mailbox: Mailbox) async throws -> ExternalMailInfo {
+        try await perform(request: authenticatedRequest(.externalMailFlag(
+            hostingId: mailbox.hostingId,
+            mailboxName: mailbox.mailbox
+        )))
+        .data
```

Comment:
We don't need `.data` anymore, you can safely remove it 😊

It’s thanks to a recent PR from Philippe on Core ! If you want to take a look, it's the function `perform<T: Decodable>(request: DataRequest, decoder: JSONDecoder = ApiFetcher.decoder) async throws -> T` on `InfomaniakCore.ApiFetcher.swift:153`